pypsa/io.py

```diff
@@ -386,6 +386,12 @@ def import_from_csv_folder(network, csv_folder_name, encoding=None, skip_time=Fa
         <https://docs.python.org/3/library/codecs.html#standard-encodings>`_
     skip_time : bool, default False
         Skip reading in time dependent attributes
+
+    Examples
+    ----------
+    >>> import_from_csv(network,csv_folder_name)
+    OR
+    >>> network.import_from_csv(csv_folder_name)
```

Comment:
I'd keep only the second form as a method on `network`

Reply:
For some reason, now the `Examples` vanished from https://pypsa-readthedocs.readthedocs.io/en/readthedocs/import_export.html . Do you know why?